kratos/spatial_containers/geometrical_objects_bins.h

```diff
@@ -79,8 +79,8 @@ class KRATOS_API(KRATOS_CORE) GeometricalObjectsBins
             for (TIteratorType i_object = GeometricalObjectsBegin ; i_object != GeometricalObjectsEnd ; i_object++){
                 mBoundingBox.Extend(i_object->GetGeometry().begin() , i_object->GetGeometry().end());
             }
+            mBoundingBox.Extend(Tolerance);
```

Comment:
Shouldn't the default tolerance of this function be `std::numeric_limits<double>::epsilon()` instead of `1e-12` then?

Aside from that all ok. I approve because its just a question.

Reply:
It is the current tolerance, better don't touch to avoid break things

Reply:
:+1: